elastic_transport/_async_transport.py

```diff
@@ -185,6 +185,8 @@ async def perform_request(  # type: ignore[override,return]
         retry_on_timeout: Union[bool, DefaultType] = DEFAULT,
         request_timeout: Union[Optional[float], DefaultType] = DEFAULT,
         client_meta: Union[Tuple[Tuple[str, str], ...], DefaultType] = DEFAULT,
+        endpoint_id: Union[str, DefaultType] = DEFAULT,
```

Comment:
If you are going to need other specific parameters maybe it makes sense to put them into an OTelContext (or whatever) class and send that instead of separated configurations

Reply:
This is a great point, and I wondered about it, given the already large number of parameters and potential breaking change in the future. However, I decided against it:

 * https://en.wikipedia.org/wiki/Rule_of_three_(computer_programming) does not really apply but note that we only have two parameters here
 * Flat is better than nested
 * I can't predict how the Elasticsearch semantic conventions will evolve, but I don't need any other parameters right now, and it's not that high effort to switch to an OTelContext class in the future if needed.